css/pages/events.scss

```diff
@@ -12,50 +12,44 @@
     background-color:none;
```

Comment:
You can delete this whole file and remove the import from _index.scss since you are not using them anymore :)